samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -283,12 +288,25 @@ Building and running
 .. include:: /includes/build_and_run.txt
 
 .. note::
-   On the nRF5340 development kit, the Direct Test Mode sample is a standalone network sample.
-   It does not require any counterpart application sample.
-   However, you must still program the application core to boot up the network core.
-   You can use any sample for this, for example, the :ref:`nrf5340_empty_app_core`.
-   The :ref:`nrf5340_empty_app_core` is built and programmed automatically by default.
-   If you want to program another sample for the application core, unset the :kconfig:option:`CONFIG_NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE` option.
+   On the nRF5340 development kit, the Direct Test Mode sample requires :ref:`nrf5340_remote_shell` sample on the application core.
+   The :ref:`nrf5340_remote_shell` is built and programmed automatically by default.
+   If you want to program your custom solution for the application core, unset the :kconfig:option:`CONFIG_NCS_SAMPLE_REMOTE_SHELL_CHILD_IMAGE` option.
```

Comment:
```suggestion
   On the nRF5340 development kit, this sample requires the :ref:`nrf5340_remote_shell` sample on the application core.
   The Remote IPC shell sample is built and programmed automatically by default.
   If you want to program your custom solution for the application core, unset the :kconfig:option:`CONFIG_NCS_SAMPLE_REMOTE_SHELL_CHILD_IMAGE` Kconfig option.
```
Took out the other link because we just linked to the same in the previous sentence.